protocols.csv

```diff
@@ -10,4 +10,5 @@ code	size	name
 421	V	ipfs
 480	0	http
 443	0	https
-444	10	onion
\ No newline at end of file
+477	0	websockets
```

Comment:
having `ws` mean "path, but not encapsulated" makes it next to useless for us. Ipfs can't use a multiaddr that it cant encapsulate. The only distinction that makes sense is for `ws` to mean "No path" and `wsn` to be the number thing as we said.